test/products/searchProductsIT.js

```diff
@@ -133,7 +133,7 @@ describe('magento searchProducts', function() {
                     expect(product).to.have.own.property('createdAt');
 
                     expect(product.variants).to.have.lengthOf(15);
-                    expect(product.attributes).to.have.lengthOf(2);
+                    expect(product.attributes).to.have.lengthOf(4);
```

Comment:
That has nothing to do with the changes we did to the images, I think this either comes from a bug or from something that shouldn't have changed ... I checked the result of getting that product, and the base product has the attributes `color` and `size`, so this is totally wrong. We should understand where this is coming from, but the it's wrong to fix that test.

Reply:
@cjelger then it's an issue somewhere in the code because the CIF query returns four attributes for the base product - color, size, features, summary